src/app/services/user.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@angular/core';
+import { User } from '../interfaces/user';
+import { BehaviorSubject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class UserService {
+  constructor() {}
+
+  users$: BehaviorSubject<User[]> = new BehaviorSubject([
+    { firstName: 'Mike', lastName: 'Wheeler', id: '2sdns2fqwdhzc' },
+    { firstName: 'Dustin', lastName: 'Henderson', id: '1sdns2fqwdhzc' },
+    { firstName: 'Lucas', lastName: 'Sinclair', id: '4sdns2fqwdhzc' },
+    { firstName: 'Will', lastName: 'Byers', id: '3sdns2fqwdhzc' },
+  ]);
```

Comment:
The standard way is to make the BehaviourSubject private and only accessible on the component itself and make a public Observable for this BehaviourSubject.